Framework/DataHandling/src/LoadILLReflectometry.cpp

```diff
@@ -540,7 +547,19 @@ std::vector<double> LoadILLReflectometry::getXValues() {
         g_log.error() << "First chopper velocity " << chop1Speed
                       << ". Check you NeXus file.\n";
       }
-      const double chopWindow = 45.0;
+
```

Comment:
Forgive me for these questions but this is very out of my depth! What are we chopping here? Why are we chopping it? What is the significance of cycle 203 in terms of chopping, and what does IPF and IDF mean?

Reply:
The reactor sources are continuous, so the beam has to be chopped to create pulses. It is the reactor cycle 203, when the format of the nexus file has changed. IPF and IDF stand for instrument parameter file and instrument definition file.